### PR TITLE
ROC-5850 DQ confirmation page

### PR DESCRIPTION
**Pull request by robertparkinson**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-5850

### Change description

Update part admission claim details for DQs and mediation

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/app/claims/models/claim.ts [hunk 1]
```diff
@@ -532,7 +534,12 @@ export class Claim {
   }
 
   private hasClaimantRejectedPartAdmission (): boolean {
-    return this.claimantResponse && this.claimantResponse.type === ClaimantResponseType.REJECTION
+    return !ClaimFeatureToggles.isFeatureEnabledOnClaim(this, 'directionsQuestionnaire') && this.claimantResponse && this.claimantResponse.type === ClaimantResponseType.REJECTION
+      && this.response.responseType === ResponseType.PART_ADMISSION
+  }
+
+  private hasClaimantRejectedPartAdmissionDQs (): boolean {
+    return ClaimFeatureToggles.isFeatureEnabledOnClaim(this, 'directionsQuestionnaire') && this.claimantResponse && this.claimantResponse.type === ClaimantResponseType.REJECTION
```
Comment (KiranVarma02) [inline src/main/app/claims/models/claim.ts]: above two methods are same except one returns true and other false. Can we put both in 1 method (if-else-block) and change method name to something more appropriate like `hasClaimantRejectedPartAdmissionWithDQ` , just a suggestion!!

Comment (robertparkinson) [reply src/main/app/claims/models/claim.ts]: I'm not sure I understand your point? can you come explain what you mean to me?

Comment (KiranVarma02) [reply src/main/app/claims/models/claim.ts]: i guess its not an issue if we are removing this logic when DQ is enabled in Production.

File: src/test/common/components/free-mediation/freeMediationTask.ts [hunk 2]
```diff
@@ -75,4 +75,12 @@ describe('Free mediation task', () => {
 
     expect(FreeMediationTask.isCanWeUseCompleted(mediationDraft)).to.be.true
   })
+
+  it('isMediationDisagreementCompleted should be completed when willYouTryMediation and mediationDisagreement are set to not', () => {
```
Comment (KiranVarma02) [inline src/test/common/components/free-mediation/freeMediationTask.ts]: set to Not or set to No?

Comment (robertparkinson) [reply src/test/common/components/free-mediation/freeMediationTask.ts]: updated the name of the test.

**Pull request merged**